mechanism2d/robot.py

```diff
@@ -0,0 +1,57 @@
+import wpilib
+
+# This sample program shows how to use Mechanism2d - a visual representation of arms, elevators,
```

Comment:
Generally non-license comments at the top of a file should be in docstring form? Everything else looks great, thanks!